src/quickpicks/build/ios.ts

```diff
@@ -3,15 +3,29 @@ import { Target } from '../..//types/cli';
 import { IosCert, IosCertificateType, IosProvisioningType, ProvisioningProfile } from '../../types/common';
 import { quickPick, CustomQuickPick } from '../common';
 import { deviceQuickPick, DeviceQuickPickItem } from './common';
-
+import * as vscode from 'vscode';
 export async function selectiOSCertificate (buildType: string): Promise<IosCert> {
 	const certificateType: IosCertificateType = buildType === 'run' ? 'developer' : 'distribution';
-	const certificates = ExtensionContainer.environment.iOSCertificates(certificateType).map(cert => ({
-		description: `Expires: ${new Date(cert.after).toLocaleString('en-US')}`,
-		label: cert.fullname,
-		pem: cert.pem,
-		id: cert.fullname
-	}));
+	let organizationName = '';
+	const certificates: any[] = [];
+	const configs = vscode.workspace.getConfiguration('titanium.ios');
+	const organizations = JSON.parse(configs.organizations);
+	ExtensionContainer.environment.iOSCertificates(certificateType).forEach(cert => {
+		let organization = cert.name.split('(')[1];
```

Comment:
I unfortunately don't have any iOS certs to test, but I'm curious if the value we're extracting from the brackets is available in any of the data [ioslib sets](https://github.com/tidev/ioslib/blob/master/src/certs.js#L58-L67), I think the type we use in here is partially implemented to just what we use.

If you run `ti info -t ios -o json` you can check to see if it's possible to use it and then update the [type to include it](https://github.com/tidev/vscode-titanium/blob/eeff82e7a411c6df170223e37f40b91d6f44f1fe/src/types/common.ts#L10-L18)